source/getting-started/installation-raspberry-pi.markdown

```diff
@@ -1,29 +1,151 @@
+
+
+### {% linkable_title Raspberry Pi Camera %}
+The Raspberry Pi Camera is a specific camera for the Raspberry Pi boards. For more information about the camera I suggest reading the [documenation](https://www.raspberrypi.org/documentation/usage/camera/) from the Raspberry Pi foundation.  
+
+To use the camera it needs to be enabled with the `raspi-config` utility.
+```bash
+$ sudo raspi-config
+```
+Select `Enable camera` choose `<Yes>` and hit `Enter`, then go to `Finish` and you'll be prompted to reboot.  
+After reboot add your `homeassistant` account to the `video` group.
+```bash
+$ sudo adduser homeassistant video
+```
+After this follow the [Raspberry Pi Camera component](/components/camera.rpi_camera/) page.
 
- - Install Home Assistant
- - Launch Home Assistant and serve the web interface on [http://localhost:8123](http://localhost:8123)
+### {% linkable_title Raspberry Pi GPIO %}
+Each of the following devices are connected to the GPIO pins on the Raspberry Pi.
+For more details about the GPIO layout, visit the Wikipedia [article](https://en.wikipedia.org/wiki/Raspberry_Pi#GPIO_connector) about the Raspberry Pi.  
+Since these are not normaly used some extra permission will need to be added.
+In general the permission that is needed is granted by adding the `homeassistant` account to the `gpio` group.
 
-There is also a [video tutorial](https://www.youtube.com/watch?v=GjzOXkPb7XE) created by [brusc](https://github.com/brusc).
+
+### {% linkable_title Raspberry Pi Cover %}
+Add your `homeassistant` account to the `gpio` group
+```bash
+$ sudo adduser homeassistant gpio
+```
+After this follow the [Raspberry Pi Cover component](/components/cover.rpi_gpio/) page.
+
+### {% linkable_title DHT Sensor %}
+Add your `homeassistant` account to the `gpio` group
+```bash
+$ sudo adduser homeassistant gpio
+```
+After this follow the [DHT Sensor component](/components/sensor.dht/) page.
+
+
+### {% linkable_title Raspberry PI GPIO Binary Sensor %}
+Add your `homeassistant` account to the `gpio` group
+```bash
+$ sudo adduser homeassistant gpio
+```
+After this follow the [Raspberry PI GPIO Binary Sensor component](/components/binary_sensor.rpi_gpio/) page.
+
+### {% linkable_title Raspberry PI GPIO Switch %}
+Add your `homeassistant` account to the `gpio` group.
+```bash
+$ sudo adduser homeassistant gpio
+```
+After this follow the [Raspberry PI GPIO Switch component](/components/switch.rpi_gpio/) page.
+
+### {% linkable_title Raspberry Pi RF Switch %}
+Add your `homeassistant` account to the `gpio` group
+```bash
+$ sudo adduser homeassistant gpio
+```
+After this follow the [Raspberry Pi RF Switch component](/components/switch.rpi_rf/) page.
+
+### {% linkable_title One wire Sensor %}
+The One wire sensor requires that support for it is enabled on the Raspberry Pi and that the One Wire device is connected to GPIO pin 4.
```

Comment:
The One wire sensor requires support for it to be enabled on the Raspberry Pi...